apps/dashboard/src/components/workflow-editor/url-input.tsx

```diff
@@ -49,7 +49,11 @@ export const URLInput = ({
                   <FormControl>
                     {asEditor ? (
                       <Editor
-                        asInput
+                        singleLine
+                        indentWithTab={false}
```

Comment:
Since most of Editors don't allow indentWithTab, just set it to false by default.